src/common/language_dropdown.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery'
+
+export default class LanguageDropdown {
```

Comment:
:smile: you're right, needed to wrap it in another click event as such:
`    $('.language')
      .click(function onClick() {
        $('ul#select_language li')
          .click(function onClick() {
            const lang = $(this).attr('class')
            document.location.search = `l=${lang}`
          })
      })`